### ISSUE TITLE
Native iOS view inside scrollview still visible

### ISSUE THREAD

**Issue by timobaehr** [flutter/flutter#78205]
I encounted this bug inside iOS native views (no issue on Android!). To keep it short and simple I simply use the `webview_flutter` plugin to demonstrate the issue:

```dart
void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        body: SingleChildScrollView(
            child: Column(
              children: [
                Container(
                  height: 1000,
                  color: Colors.black26,
                  child: Center(
                    child: Text('Placeholder'),
                  ),
                ),

                SizedBox(
                  height: 200,
                  child: WebView(
                    initialUrl: 'https://flutter.dev',
                  ),
                ),
              ],
            )
        ),
      ),
    );
  }
}
```

If I first scroll down, the native view is properly rendered. If I then scroll up **very fast**, the native iOS view is still visible.
![Simulator Screen Shot - iPhone 12 Pro Max - 2021-03-15 at 12 46 03](https://user-images.githubusercontent.com/13302336/111149033-a54af200-858c-11eb-9a07-e9ecf653080e.png)

I encountered this bug when creating my own iOS view with `UiKitView`. So bug is not related to the webview_flutter plugin I used here for demonstration. I only attached the screenshot where the view is hanging around at the bottom, but same can happen at the top. I tested it both on iOS simulator and also a real device.

Output of `flutter doctor -v`:
```
[✓] Flutter (Channel stable, 2.0.1, on macOS 11.2.2 20D80 darwin-x64, locale de-DE)
    • Flutter version 2.0.1 at /Users/timo/flutter
    • Framework revision c5a4b4029c (11 days ago), 2021-03-04 09:47:48 -0800
    • Engine revision 40441def69
    • Dart version 2.12.0

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/timo/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.54.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.20.0

[✓] Connected device (1 available)
    • iPhone 12 Pro Max (mobile) • 8C1C1834-2E41-497F-968F-1E780DC4C4FB • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-4
      (simulator)
```

**Comment by timobaehr** [comment]
Suggested labels: `platform-ios`, `f: scrolling`, `has reproducible steps`

**Comment by dzhaparoff** [comment]
Same issue with google_maps_flutter and others plugins. Native view remains visible under other layer in stack, or after very quick scroll. This issue present in flutter >= 2.0 and not in older releases.

**Comment by NickJokic** [comment]
**+1** (google_maps_flutter)

**Comment by yasargil** [comment]
I have already opened an issues see:
https://github.com/flutter/flutter/issues/76097

**Comment by timobaehr** [comment]
Solved with Flutter 2.2. stable release
